src/WatchViewHelper.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+require_once 'src/Watch.php';
+
+class WatchViewHelper
```

Comment:
The two methods in here will need unit tests. It doesn't look like you are using displaySingleWatch, so you should consider deleting that so you don't have to write unit tests for it